index.js

```diff
@@ -92,6 +90,7 @@ const ORDERED_DEFAULTS = {
       'resolve',
       'json',
       'commonjs',
+      'externalGlobals',
```

Comment:
Seems like we should be adding this plugin to the `browser` dist only. The module dist should keep the require/import.

Reply:
Yup, it's only getting added here, for the browser builds.

Reply:
ahh your right, the way that github showed the diff let me to believe it was under module.